From 29281b874e6d24a347151cc817ec0dd020b87431 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
Date: Thu, 30 Oct 2008 23:27:01 -0400
Subject: [PATCH] markers-remove-old-comment

Marers remove old comment

> > + * Note : the empty asm volatile with read constraint is used here instead of a
> > + * "used" attribute to fix a gcc 4.1.x bug.
>
> There seems no empty asm...

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
---
 include/linux/marker.h |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/include/linux/marker.h b/include/linux/marker.h
index 7c65181..9057047 100644
--- a/include/linux/marker.h
+++ b/include/linux/marker.h
@@ -71,8 +71,6 @@ struct marker {
 		_DEFINE_MARKER(name, #tp_name, tp_cb, format)
 
 /*
- * Note : the empty asm volatile with read constraint is used here instead of a
- * "used" attribute to fix a gcc 4.1.x bug.
  * Make sure the alignment of the structure in the __markers section will
  * not add unwanted padding between the beginning of the section and the
  * structure. Force alignment to the same alignment as the section start.
-- 
1.5.5.1

